packages/ODEs/src/scripting-tools.ts

```diff
@@ -1074,6 +1092,34 @@ function getSolutionDfColsNames(ivp: IVP): string[] {
   return res;
 }
 
+/** Check solver settings */
+function checkSolverSettings(line: string): void {
+  const settings = new Map<string, string>();
+
+  const openBraceIdx = line.indexOf(BRACE_OPEN);
+  const closeBraceIdx = line.indexOf(BRACE_CLOSE);
+  let sepIdx: number;
+
+  if (openBraceIdx >= closeBraceIdx)
+    throw new Error(`${ERROR_MSG.BRACES}. Check the line '${line}'`);
+
+  line.slice(openBraceIdx + 1, closeBraceIdx).split(ANNOT_SEPAR).forEach((item: string) => {
```

Comment:
I know it's minor, but still, let's prefer the canonical for loop instead of forEach, it's easier to read (also a little bit faster)

```
for (const item: string of  line.slice(openBraceIdx + 1, closeBraceIdx).split(ANNOT_SEPAR)) {
   ...
```

Reply:
Done